salt/utils/minions.py

```diff
@@ -643,16 +647,19 @@ def connected_ids(self, subset=None, show_ipv4=False, include_localhost=False):
                     continue
                 grains = mdata.get('grains', {})
                 for ipv4 in grains.get('ipv4', []):
-                    if ipv4 == '127.0.0.1' and not include_localhost:
-                        continue
-                    if ipv4 == '0.0.0.0':
-                        continue
                     if ipv4 in addrs:
                         if show_ipv4:
                             minions.add((id_, ipv4))
                         else:
                             minions.add(id_)
                         break
+                for ipv6 in grains.get('ipv6', []):
+                    if ipv6 in addrs:
+                        if show_ipv4:
```

Comment:
This is request to show IPv4, and still you putting IPv6. I guess, this parameter needs either be renamed to "show_ip" (preferred) or we need to add another one (no idea why). In fact, this parameter/option is only used once, in manage runner, list_state function.